pkg/controller/controller_utils.go

```diff
@@ -86,6 +87,11 @@ var UpdateTaintBackoff = wait.Backoff{
 	Jitter:   1.0,
 }
 
+var ShutdownTaint = &v1.Taint{
```

Comment:
should be in `pkg/controller/nodelifecycle/node_lifecycle_controller.go`

Reply:
I do not fully agree with this, because then we need import nodelifecycle package to https://github.com/zetaab/kubernetes/blob/b900bf3ed391efd774237d491bdf50780057941e/pkg/controller/cloud/node_controller.go or then make same variable there? Which one is better 1) import nodelifecycle package to cloudcontroller (the worst option imo) 2) make twice same variable 3) keep it as is (in controller_utils)

Reply:
@k82cn - I’d prefer keeping this here for now, I’ll open a dedicated PR to move all the taints to the same place.

Reply:
that's ok to me :)